specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -6925,6 +6925,101 @@
       },
       "description": "Base type for skills."
     },
+    "CustomEntity": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The top-level entity descriptor. Matches in the skill output will be grouped by this name, and it should represent the \"normalized\" form of the text being found."
+        },
+        "description": {
+          "type": "string",
+          "description": "This field can be used as a passthrough for custom metadata about the matched text(s). The value of this field will appear with every match of its entity in the skill output."
+        },
+        "type": {
+          "type": "string",
+          "description": "This field can be used as a passthrough for custom metadata about the matched text(s). The value of this field will appear with every match of its entity in the skill output."
+        },
+        "subtype": {
+          "type": "string",
+          "description": "This field can be used as a passthrough for custom metadata about the matched text(s). The value of this field will appear with every match of its entity in the skill output."
+        },
+        "id": {
+          "type": "string",
+          "description": "This field can be used as a passthrough for custom metadata about the matched text(s). The value of this field will appear with every match of its entity in the skill output."
+        },
+        "caseSensitive": {
+          "type": "boolean",
+          "x-nullable": true,
```

Comment:
There are a lot of x-nullable fields. Can you just not send them when null? Is there an advantage to sending, for example, `"caseSensitive": null` and similar for everything that isn't explicitly set?